frontend/src/vendor/profile/contacts/VendorContacts.js

```diff
@@ -71,27 +71,31 @@ const VendorContacts = (props) => {
     },
   ];
 
-  return (
-    <VendorPadding>
-      <ContactTitle>Elérhetőségek</ContactTitle>
-      {contacts.map((contact, index) => {
-        return (
-          contact.title && (
-            <ContactInfo key={v4()}>
-              <ContactLogo src={contact.logo} alt="" />
-              <ContactLinks
-                target="_blank"
-                rel="noopener noreferrer external"
-                href={contact.link + contact.title}
-              >
-                {contact.title}
-              </ContactLinks>
-            </ContactInfo>
-          )
-        );
-      })}
-    </VendorPadding>
-  );
+  if (contacts.some((contact) => contact.title)) {
```

Comment:
If helyett lehetne &&-el csinálni, mivel az else ág úgy is lényegtelen:

```
return contacts.some((contact) => contact.title) && <VendorPadding> ...
```

vagy olvashatóbban egy változóba kiemelni

```
const hasContact = contacts.some((contact) => contact.title)

return hasContact && <VendorPadding> ...
```